src/utils/user-role-utils.tsx

```diff
@@ -0,0 +1,33 @@
+import { useAtomValue } from "jotai";
+import { authAtom } from "../atoms/auth";
+import { useLocation } from "react-router";
+
+/**
+ * User Role Utils class
+ */
+export default class UserRoleUtils {
+  /**
+   * Check if the logged-in user has an admin role
+   *
+   * @returns boolean
```

Comment:
What does it return?

Reply:
boolean, indicates if user is admin and in admin route